include/boost/gil/extension/toolbox/color_spaces/ycbcr.hpp

```diff
@@ -55,6 +56,21 @@ typedef boost::gil::layout<ycbcr_709__t> ycbcr_709__layout_t;
 GIL_DEFINE_ALL_TYPEDEFS(8, uint8_t, ycbcr_601_)
 GIL_DEFINE_ALL_TYPEDEFS(8, uint8_t, ycbcr_709_)
 
+namespace detail {
+
+// Source:boost/algorithm/clamp.hpp
+template<typename T>
+BOOST_CXX14_CONSTEXPR T const& clamp(T const& val,
```

Comment:
Does this have to be constexpr ? Or, to put it differently, is it worth introducing a C++14-only requirement here ?

Reply:
Hmm, good question. Perhaps C++11 `constexpr` will be sufficient. 

Unfortunately, toolbox tests are currently failing to compile with GCC for me, so I'll have to look at those issues first.

Reply:
I've replaced `BOOST_CXX14_CONSTEXPR` with `constexpr`.

Reply:
Excellent, thanks !